### PR TITLE
feat: hide and seek

### PR DESCRIPTION
**Pull request by 55Honey**
This PR adds an automated hide and seek event.

For testing:
Run the hideAndSeek_up.sql against your world db.
Type `.hideandseek 1` to start the demo event.
`.tele durotar` takes you to the appx position of the target object for testing.

### CODE REVIEW

File: customWorldboss.lua [hunk 1]
```diff
@@ -819,8 +819,12 @@ local function eS_getEncounterDuration()
 end
 
 local function eS_getTimeSince(time)
-    local dt = GetTimeDiff(time)
-    return dt
+    if time then
+        local dt = GetTimeDiff(time)
+        return dt
+    else
+        return 0
```
Comment (Helias) [inline customWorldboss.lua:826]: could you write it in the following way?
```suggestion
    if time then
        return GetTimeDiff(time)
    end

    return 0
```

normally when you use if - else and both return, you don't need to write the else

Comment (55Honey) [reply customWorldboss.lua:826]: Looks neat, I'll try!

Comment (Helias) [reply customWorldboss.lua:826]: if it works, do it also in other part of code

Comment (55Honey) [reply customWorldboss.lua]: I'd rather not spend too much time on a script's code style which is due for a full rewrite anyway.

**Pull request merged**